app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -96,7 +96,7 @@ description: |
   echo "Starting the subscription-manager registration process"
 
   # Set up subscription-manager
-  <%= snippet("subscription_manager_setup", variables: { subman_setup_scenario: 'provisioning' }).strip -%>
+  <%= indent(2) { snippet("subscription_manager_setup", variables: { subman_setup_scenario: 'provisioning' }).strip } -%>
```

Comment:
Can you try to set skip1 as described in https://github.com/theforeman/foreman/blob/e49650aa2f262f562efa3b024449021119699200/app/services/foreman/renderer/scope/macros/base.rb#L135 

If you have a look at https://github.com/theforeman/foreman/pull/10200/files#diff-2123697b8041a8f9d6efe3585f582d0052a40494914c861ab7b9bcb7bd0af0baR41 and the following line, the indention is off by 2 spaces